doc/Extensions/Applications.md

```diff
@@ -2179,6 +2179,71 @@ extend supervisord /etc/snmp/supervisord.py
 systemctl restart snmpd
 ```
 
+## Sagan
+
+For metrics the stats are migrated as below from the stats JSON.
+
+'f_drop_percent' and 'drop_percent' are computed based on the found data.
+
+```
+uptime              => $json->{stats}{uptime},
```

Comment:
This format is very odd and a little confusing.

Are you saying the application output json and you are changing that json to a different format?  Perhaps that would be better included in the server side script as a comment.

Reply:
Yeah, basically just copied it directly from the perl code for the extend.

I'll update it to the manner used by jq.

Reply:
@murrant  And done! That should make it clear now. :)